### PR TITLE
Handle LCD

### PR DESCRIPTION
**Pull request by marcinzzz**

### CODE REVIEW

**Comment by marcinzzz** [comment]
image

File: controllers/src/userController/Controllers.cpp [hunk 1]
```diff
@@ -36,6 +37,13 @@ int Controllers::getFormatedFreq()
   return round10(freq);
 }
 
+void Controllers::displayFreq() {
+  char buffer[6];
+  int freq = Controllers::getFormatedFreq();
+  sprintf(buffer, "%3d.%02d", freq / 100, freq % 100);
```
Comment (Zummek) [inline controllers/src/userController/Controllers.cpp]: Hmm What about simple divide freq by 100 and assign it to float or better, char? There would always be 2 decimal places(i'm not sure now)

Is it possible to add immediately ` MHz` in sprintf string?

File: controllers/src/userController/main.cpp [hunk 2]
```diff
@@ -14,11 +15,15 @@ void setup()
...
   AppRadio::init();
+
+  Lcd::init();
 }
 
 void loop()
 {
   AppRadio::radio.checkRDS();
   Controllers::readAndProcess();
+  Controllers::displayFreq();
```
Comment (Zummek) [inline controllers/src/userController/main.cpp]: I suggest refreshing values on display only on changes. Implement function like this in AppRadio methods, next to Serial.printfs

this gives uC more time for other things like reading RDS

Comment (Zummek) [reply controllers/src/userController/main.cpp]: Following this, I think it's worth it to move all display functions to LCD class

**Review by Zummek — COMMENTED** [review]
Well, 
but I added some suggestions about optimization

Add to schematic model of I2C converter as value of component, we need to know this model

PS make rebase ;)

File: controllers/src/userController/main.cpp (hunk 2 above)
Comment (Zummek) [reply controllers/src/userController/main.cpp]: But if procedure require e.g two lines I think it shouldn't be moved to separated function

File: controllers/src/userController/Controllers.cpp (hunk 1 above)
Comment (marcinzzz) [reply controllers/src/userController/Controllers.cpp]: ### g++
```
int getFormatedFreq() {
    return 8890;
}

int main() {
    char buffer[11];
    int i = getFormatedFreq();
    float f = (float) i / 100;
    sprintf(buffer, "%6.2f MHz", f);
    printf("%s\n", buffer);
    return 0;
}
```
Result
```
 88.90 MHz
```
### Arduino
```
void Controllers::displayFreq() {
  char buffer[11];
  int freq = Controllers::getFormatedFreq();
  float f = (float) freq / 100;
  sprintf(buffer, "%6.2f", f);
  Lcd::write(0, 0, buffer);
}
```
Result
```
     ?
````
I'm going to leave it with ```freq / 100```;```freq % 100``` but add ```MHz``` to buffer.

**Comment by marcinzzz** [comment]
There is problem with a name of I<sup>2</sup>C converter, best I can do is to write chip name or that it is compatible with HD44789 driver.

File: controllers/src/userController/Controllers.cpp (hunk 1 above)
Comment (Zummek) [reply controllers/src/userController/Controllers.cpp]: 😆  okej

**Review by Zummek — APPROVED** [review]

**Pull request merged**